src/components/pages/Contacts/Contacts.tsx

```diff
@@ -0,0 +1,28 @@
+import InfoBlock from '../../InfoBlock/InfoBlock';
+import SectionTitle from '../../common/SectionTitle';
+
+import sectionImg from '../../../assets/img/sections/contacts.png'
+import React from "react";
+
+const Contacts: React.FC = () => {
+  return (
+    <InfoBlock sectionImg={sectionImg}>
+      <div>
+      <SectionTitle>Контакти</SectionTitle>
+        <div className='mt-4'>
+          <div className='leading-6'>
+            <div className="font-semibold mb-1">Інформаційна підтримка</div>
+            <div>тел.: <a href='tel:380444549845'>+38 (044) 454 98 45</a></div>
+            <div>факс: <a href='tel:380444549845'>+38 (044) 454 98 45</a></div>
```

Comment:
```suggestion
            <div>тел.: <a href='tel:380444549845'>+380 (44) 454 98 45</a></div>
            <div>факс: <a href='tel:380444549845'>+380 (44) 454 98 45</a></div>
```

Reply:
Detailed information - 
https://uk.wikipedia.org/wiki/%D0%A2%D0%B5%D0%BB%D0%B5%D1%84%D0%BE%D0%BD%D0%BD%D1%96_%D0%BA%D0%BE%D0%B4%D0%B8_%D0%A3%D0%BA%D1%80%D0%B0%D1%97%D0%BD%D0%B8